cmd/cli/mesh_list.go

```diff
@@ -62,17 +63,53 @@ func (l *meshListCmd) run() error {
 	}
 
 	w := newTabWriter(l.out)
-	fmt.Fprintln(w, "MESH NAME\tNAMESPACE\t")
+	nds, _ := l.clientSet.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
```

Comment:
> I have a consider here, is there different to `list.Items[0].ClusterName `? Why we should search it from the cache rather than directly use the `list` variable defined before?
I tried it but it was not working on minikube and kind.

Reply:
@draychev  Hello, sir. Any idea here?

Reply:
Hi @mudit-01, Is that work on Kind?

Reply:
Hi @Aisuko yes I tried it on minikube and kind it worked on both.